app/controllers/users_controller.rb

```diff
@@ -29,7 +29,7 @@ def show
               find_user(session[:user].id)
             end
 
-    @all_ontologies = LinkedData::Client::Models::Ontology.all(ignore_custom_ontologies: true)
+    @all_ontologies = LinkedData::Client::Models::Ontology.all(ignore_custom_ontologies: true).map {|x| ["#{x.name} (#{x.acronym})", x.acronym]}
```

Comment:
this code, is repeated two times, and you have already a helper that do the same 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/e5f77936-033a-401e-8f99-ca732b86619b)

Reply:
add an argument to `onts_for_select` to ignore or not custom_ontologies

Reply:
in all cases I need to a call for /ontologies here and this ignore or not custom_ontologies will be used only here and for this specific feature so I don't see it a good idea to add it as param.